xml/System.IO/File.xml

```diff
@@ -2422,7 +2422,7 @@
 
 This method works across disk volumes, and it does not throw an exception if the source and destination are the same.
 
-Note that if you attempt to replace a file by moving a file of the same name into that directory, an <xref:System.IO.IOException> is thrown. For that purpose, call <xref:System.IO.File.Move(System.String,System.String,System.Boolean)> instead.
+Note that if you attempt to replace a file by moving a file of the same name into that directory, an <xref:System.IO.IOException> is thrown. You can avoid this problem by calling <xref:System.IO.File.Delete(System.String)> which will only delete the file if it exists.
```

Comment:
This method exists but only in .NET Core.

@JeremyKuhne @carlossanlop should we note how you can do that in Framework vs Core here?

Reply:
The proposed suggestion by @maillouxc is the correct one for .NET Framework. We should keep the original suggestion but specify that it's only available in .NET Core 3.0 and above.

```suggestion
Note that if you attempt to replace a file by moving a file of the same name into that directory, an <xref:System.IO.IOException> is thrown.

In .NET Core 3.0 and above, you can avoid this problem by calling <xref:System.IO.File.Move(System.String,System.String,System.Boolean)> setting the parameter `overwrite` to `true`, which will replace the file if it exists.

In all .NET versions, you can call <xref:System.IO.File.Delete(System.String)> before calling `Move`, which will only delete the file if it exists.

```

Reply:
I made a minor modification to your suggestion @carlossanlop. I hope you don't mind!

Reply:
@maillouxc feel free to apply the suggestion whenever you get a chance, so we can get it merged.